test/Microsoft.ML.Tests/Scenarios/Api/AspirationalExamples.cs

```diff
@@ -60,5 +60,97 @@ public void FirstExperienceWithML()
                 PetalLength = 5.1f
             });
         }
+
+
+        [Fact]
+        public void SimpleIrisDescisionTrees()
+        {
+            var env = new ConsoleEnvironment();
+            string dataPath = "iris.txt";
+            // Create reader with specific schema.
+            var dataReader = TextLoader.CreateReader(env, dataPath,
+                ctx => (
+                    label: ctx.LoadText(0),
+                    sepalWidth: ctx.LoadFloat(1),
+                    sepalLength: ctx.LoadFloat(2),
+                    petalWidth: ctx.LoadFloat(3),
+                    petalLength: ctx.LoadFloat(4)
+                ));
+
+            var pipeline = dataReader.MakeEstimator()
+                .Append(row => (
+                    // Convert string label to key.
+                    label: row.label.Dictionarize(),
+                    // Concatenate all features into a vector.
+                    features: row.sepalWidth.ConcatWith(row.sepalLength, row.petalWidth, row.petalLength)))
+                .Append(row => (
+                    label: row.label,
+                    prediction: row.label.PredictFastTreeMultiClass(row.features)));
+
+            // Load the data into the system.
+            var data = dataReader.Read(dataPath);
+            var model = pipeline.Fit(data);
+
+            var predictions = model.Transform(dataReader.Read(testDataPath));
+            var metrics = MultiClassEvaluator.Evaluate(predictions, row => row.label, row => row.prediction);
+        }
+
+        [Fact]
+        public void TwitterSentimentAnalysis()
+        {
+            var env = new ConsoleEnvironment();
+            var dataPath = "wikipedia-detox-250-line-data.tsv";
+            // Load the data into the system.
+            var dataReader = TextLoader.CreateReader(env, dataPath,
+                    ctx => (
+                        label: ctx.LoadFloat(0),
+                        text: ctx.LoadText(1)),
+                    hasHeader: true);
+
+            var pipeline = dataReader.MakeEstimator()
+                .Append(row => (
+                    label: row.label,
+                    // Concatenate all features into a vector.
+                    features: row.text.TextFeaturizer()))
+                 .Append(row => (
+                    label: row.label,
+                    row.label.TrainLinearClassification(row.features)));
```

Comment:
Should `TextFeaturizer` be `FeaturizeText` for consistency? `TrainLinearClassification` -> `PredictWithLinearClassifier`?